### DISCUSSION TITLE
【求助】跑不过了

### DISCUSSION THREAD

**Discussion by RiceShowerOVO** [share121/inter-knot#2586] [其他]
有没有人帮我跑天皇赏春，跑完后有1600菲林作为报答.
![1E19EFF4DAA5A0262B7D3DD8CEE6B084](https://github.com/user-attachments/assets/94375ba2-6058-499f-9fcb-49cdc75d6b89)

**Comment by icaros-first** [comment]
用米浴跑吗？准备刷因子吧，还有借个好点的卡
